.github/workflows/ci-run-tests.yml

```diff
@@ -0,0 +1,27 @@
+name: Run all tests
+
+on:
+  push:
```

Comment:
I would remove the push trigger, as this would re-run the tests a second time upon merge, which is not necessary.